starter-code/functionsandarrays.js

```diff
@@ -0,0 +1,168 @@
+/*var numbers = [2, 6, 9, 10, 7, 4, 1, 9];
+
+function averageNumbers(numbers){
+  for(var i = 0; i < numbers.length; i++) {
+sumOfElements = sumOfElements + averageNumbersnumbers[i]}
+
+var averageSum = (sumOfElements / numberOfElements);
+console.log(averageSum);
+
+}
+
+var average = averageNumbers(numbers);
+console.log(average);
+// 6*/
```

Comment:
for this part, try to be sure that you are declaring and using the variables on the correct place. The logic is good